Source/CombatExtended/CombatExtended/Alerts/Alert_SmokeInhalation.cs

```diff
@@ -0,0 +1,65 @@
+﻿using RimWorld;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using UnityEngine;
+using Verse;
+
+namespace CombatExtended.CombatExtended.Alerts
+{
+    class Alert_SmokeInhalation : Alert_Critical
+    {
+        private List<Pawn> SmokePoisonedPawns
+        {
+            get
+            {
+                List<Pawn> smokePoisonedPawns = new List<Pawn>();
+                List<Pawn> colonistsPrisonersAnimals = new List<Pawn>();
+                colonistsPrisonersAnimals.AddRange(PawnsFinder.AllMapsCaravansAndTravelingTransportPods_Alive_OfPlayerFaction_NoCryptosleep);
+                colonistsPrisonersAnimals.AddRange(PawnsFinder.AllMaps_PrisonersOfColonySpawned);
+                foreach (Pawn p in colonistsPrisonersAnimals)
+                {
+                    if (p.GetStatValue(CE_StatDefOf.SmokeSensitivity) > 0f)
+                    {
+                        float smokeInhalationSeverity = p.health?.hediffSet?.GetFirstHediffOfDef(CE_HediffDefOf.SmokeInhalation, true)?.Severity ?? 0f;
+                        if (smokeInhalationSeverity > 0.4f)
+                        {
+                            smokePoisonedPawns.Add(p);
+                        }
+                    }
+                }
+                return smokePoisonedPawns;
+            }
+        }
+
+        public override string GetLabel()
+        {
+            return "CE_SmokeInhalation".Translate();
+        }
+
+        public override TaggedString GetExplanation()
+        {
+            StringBuilder stringBuilder = new StringBuilder();
+            foreach (Pawn pawn in this.SmokePoisonedPawns)
+            {
+                stringBuilder.AppendLine("  - " + pawn.NameShortColored.Resolve());
+            }
+            return "CE_SmokeInhalationDesc".Translate(stringBuilder.ToString());
+        }
+
+        public override AlertReport GetReport()
+        {
+            return AlertReport.CulpritsAre(this.SmokePoisonedPawns);
+        }
+
+        protected override Color BGColor
```

Comment:
Why are you overriding this method? Is the one from `Alert_Critical` not good enough?

Reply:
Just a personal preference, I'm not a fan of vanilla using red for everything from medical emergencies to mental break risks, so the color helps it stand out as this alert is something the player will want to do something about immediately.